eventmesh-dashboard-core/pom.xml

```diff
@@ -52,24 +58,61 @@
             <version>0.0.1-SNAPSHOT</version>
         </dependency>
 
-        <!-- meta -->
+        <!-- Meta - nacos client -->
         <dependency>
             <groupId>com.alibaba.nacos</groupId>
             <artifactId>nacos-client</artifactId>
-            <version>2.2.4</version>
+            <version>${nacos.version}</version>
         </dependency>
         <dependency>
             <groupId>io.etcd</groupId>
             <artifactId>jetcd-core</artifactId>
             <version>0.3.0</version>
         </dependency>
 
-        <!-- Event Store -->
+        <!-- health check client -->
+        <!-- EventMesh SDK -->
+        <!--        <dependency>-->
+        <!--            <groupId>org.apache.eventmesh</groupId>-->
+        <!--            <artifactId>eventmesh-sdk-java</artifactId>-->
+        <!--            <version>1.10.0-release</version>-->
+        <!--            <exclusions>-->
+        <!--                <exclusion>-->
+        <!--                    <groupId>junit</groupId>-->
+        <!--                    <artifactId>junit</artifactId>-->
+        <!--                </exclusion>-->
+        <!--                <exclusion>-->
+        <!--                    <groupId>junit</groupId>-->
+        <!--                    <artifactId>junit-dep</artifactId>-->
+        <!--                </exclusion>-->
+        <!--                <exclusion>-->
+        <!--                    <groupId>org.apache.logging.log4j</groupId>-->
+        <!--                    <artifactId>log4j-slf4j-impl</artifactId>-->
+        <!--                </exclusion>-->
+        <!--            </exclusions>-->
+        <!--        </dependency>-->
+        <!-- storage redis client -->
+        <dependency>
+            <groupId>io.lettuce</groupId>
+            <artifactId>lettuce-core</artifactId>
+        </dependency>
+        <!-- rocketmq client -->
         <dependency>
             <groupId>org.apache.rocketmq</groupId>
             <artifactId>rocketmq-client</artifactId>
-            <version>4.9.7</version>
+            <version>4.9.4</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.rocketmq</groupId>
+            <artifactId>rocketmq-tools</artifactId>
+            <version>5.2.0</version>
+        </dependency>
```

Comment:
How about  use the same version of rocketmq dependency to avoid some unpredictable conflicts and ensure compatibility

Reply:
okay, they behave the same